sycl/test/lit.cfg.py

```diff
@@ -155,12 +156,13 @@ def getDeviceCount(device_type):
     config.available_features.add('gpu')
     if cuda:
        config.available_features.add('cuda')
+       gpu_run_substitute = " env SYCL_BE=PI_CUDA "
 
     if platform.system() == "Linux":
         gpu_run_on_linux_substitute = "env SYCL_DEVICE_TYPE=GPU "
         gpu_check_on_linux_substitute = "| FileCheck %s"
         if cuda:
-            gpu_run_on_linux_substitute += " SYCL_BE=PI_CUDA "
+            gpu_run_on_linux_substitute = " env SYCL_BE=PI_CUDA "
```

Comment:
Shouldn't we have two env. variables set `SYCL_BE=PI_CUDA` and `SYCL_DEVICE_TYPE=GPU`?

Reply:
done